libraries/AP_Proximity/AP_Proximity.cpp

```diff
@@ -493,27 +367,6 @@ bool AP_Proximity::get_upward_distance(float &distance) const
     return get_upward_distance(primary_instance, distance);
```

Comment:
This will probably stop reporting upwards distance if the rangefinder doesn't become the primary sensor by luck. I'll include a fix for it